service/hub/server/config.go

```diff
@@ -83,6 +96,15 @@ func LoadConfig(fileName string) (*Config, error) {
 	if c.Server.ListenAddress == "" {
 		c.Server.ListenAddress = ":5001"
 	}
+	if c.Server.MaxRequestsPerIP == 0 {
+		c.Server.MaxRequestsPerIP = DefaultMaxRequestsPerIP
```

Comment:
can you refactor to pointer ?

Reply:
`server` refactor to a poniter?

Reply:
if server.ListenAddress == nil like this

Reply:
not necessary？